Assets/Scripts/ParticleSystemRenderer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ParticleSystemRenderer : MonoBehaviour {
```

Comment:
http://answers.unity3d.com/questions/577288/particle-system-rendering-behind-sprites.html 
According to that it's only doable through code. I think the reason to the lack of changing the rendering mode through the interface is because particle systems are usually used in 3D games, and in a 3D environment you just change the z axis position.